server/src/main/java/org/opentosca/toscana/plugins/cloudformation/visitor/CheckModelNodeVisitor.java

```diff
@@ -18,24 +18,36 @@
 import com.google.common.collect.Lists;
 
 /**
- Class for checking the models nodes
+ Class for checking the models nodes whether they are supported or not.
+ <br>
+ Methods that are not overridden will throw an {@link UnsupportedTypeException}.
  */
-public class CheckModelNodeVisitor extends CloudFormationVisitorExtension implements StrictNodeVisitor {
+public class CheckModelNodeVisitor extends CloudFormationVisitor implements StrictNodeVisitor {
 
     /**
-     Create a <tt>CheckModelNodeVisitor</tt> to check the models nodes.
+     Standard constructor.
+     <br>
+     Creates a <tt>CheckModelNodeVisitor</tt> to check the models nodes.
 
-     @param context TransformationContext to extract topology and logger
+     @param context {@link TransformationContext} to extract the topology and a logger
      */
     public CheckModelNodeVisitor(TransformationContext context) {
         super(context);
     }
 
+    /**
+     {@link Apache} node is supported.
+     */
     @Override
     public void visit(Apache node) {
         // noop
     }
 
+    /**
+     {@link Compute} node is supported.
```

Comment:
Not sure if the "X node is supported"-comments are really needed here. 
The `Only Linux, Ubuntu is supported!` part is the important one.